content-repo/gendocs.py

```diff
@@ -263,9 +325,12 @@ def main():
         f.write(index_doc_infos(playbooks_doc_infos, PLAYBOOKS_PREFIX))
         f.write("\n\n## Scripts\n\n")
         f.write(index_doc_infos(script_doc_infos, SCRIPTS_PREFIX))
+        f.write("\n\n## Release Notes\n\n")
+        f.write(index_doc_infos(release_doc_infos, RELEASES_PREFIX, headers=('Name', 'Date')))
```

Comment:
i dont really like the table to be honest, as there are only two columns with not much content in them at all.. i think a list would be better

Reply:
We can experiment with a different format on another PR. I don't want to over do it in this PR.